Neuro/Minigames/Solvers/SubmitScanSolver.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections;
+using UnityEngine;
+
+namespace Neuro.Minigames.Solvers;
+
+[MinigameSolver(typeof(MedScanMinigame))]
+
+public sealed class SubmitScanSolver : GeneralMinigameSolver<MedScanMinigame>
```

Comment:
I believe this could be an `IMinigameOpener<MedScanMinigame>`? Should double check with Alex though.

Reply:
Good point, I hadn't considered that idea, will test to verify it works as intended first.

Reply:
This does indeed work as expected, thanks for pointing it out.